docs/about-professors.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>PSB DIGIT Splash</title>
+        <meta charset="UTF-8" />
+        <meta name="author" content="Felipe Bassi" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="description"
+            content="The PSU Behrend DIGIT site, made by DIGIT students for the interested. DIGIT course info and events. Behrend Trees, data analysis, text archives, student projects, and a few galleries." />
+        <link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico?" />
+        <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P" rel="stylesheet" />
+        <link href="https://fonts.googleapis.com/css2?family=Special+Elite" rel="stylesheet" />
+        <link href="https://fonts.googleapis.com/css2?family=VT323" rel="stylesheet" />
+        <link href="https://fonts.googleapis.com/css2?family=Wallpoet" rel="stylesheet" />
+        <link rel="stylesheet" type="text/css" href="digit_style.css" />
+        <link rel="stylesheet"
+            href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" />
+        <script src="DIGIT.js"></script>
+        <script src="https://use.fontawesome.com/664cfabaeb.js"></script>
+    </head>
+    <body id="courses" class="about-professors">
+        
+        <section id="preloader"></section>
+        <section>
+            <div class="menu" id="myMenu">
+                <a href="index.html">Home</a>
+                <a href="showcase.html">Our Students</a>
+                <a href="roadmap.html">Education</a>
+                <a href="about.html" class="active">About</a>
+                <a href="ev.html">Events</a>
+                <!--                <a href="https://newtfire.org/courses/DIGITShowcase.html" target="_blank">DIGIT Works</a>-->
+                <a href="javascript:void(0);" class="icon" onclick="Menubar()">
+                    <i class="fa fa-bars"></i>
+                </a>
+            </div>
+            <div class="module">
+                <h1>About</h1>
+            </div>
+            <!--            tabbing menu-->
+            <div class="tabbingBtnsAlign">
+                <a href="https://digit-psb.github.io/DIGIT/about-students.html"
```

Comment:
These In-site links don't work because you don't have to include the https:.... The <a> tag should instead look like `<a href="about-students.html">` for example.